src/idl_gen_general.cpp

```diff
@@ -1509,11 +1529,99 @@ class GeneralGenerator : public BaseGenerator {
       code += "    return null;\n";
       code += "  }\n";
     }
+    if (lang_.language == IDLOptions::kJava)
```

Comment:
Would be nice to also output code for C#, if you have access to a C# compiler.

Reply:
Unfortunately, I am not an expert in C # and I am not familiar with the subtleties of a language. First I propose to add an alternative API for Java. For C#, do it separately.

Reply:
Fine, leave it out for now then.